src/pages/Send/Send.tsx

```diff
@@ -1,56 +1,245 @@
 import React from 'react';
+import Select from 'react-select';
+import { useGetTokensWithEgld } from 'hooks';
+import { useFormik } from 'formik';
+import { number, object, string } from 'yup';
+import { addressIsValid, formatAmount } from 'utils';
+import BigNumber from 'bignumber.js';
+import { GAS_LIMIT, GAS_PRICE, RouteNamesEnum } from 'localConstants';
+import { calculateGasLimit } from '@multiversx/sdk-dapp-form/operations/calculateGasLimit';
+import { prepareTransaction } from '@multiversx/sdk-dapp-form/hooks/useFetchGasLimit/prepareTransaction';
+import { useGetAccountInfo, useGetNetworkConfig } from 'hooks/sdkDapp.hooks';
+import { sendTransactions } from 'helpers';
+import { useNavigate } from 'react-router-dom';
+
+interface TokenOptionType {
```

Comment:
poate la types?
nu se gaseste in SendForm?

Reply:
Este mai complex